tests/Go/Functional/BaseFunctionalTest.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Go\Functional;
+
+use PHPUnit_Framework_TestCase as TestCase;
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\Process\Process;
+
+abstract class BaseFunctionalTest extends TestCase
+{
+    protected static $projectDir = __DIR__.'/../../Fixtures/project';
+    protected static $aspectCacheDir = __DIR__.'/../../Fixtures/project/var/cache/aspect';
+    protected static $consolePath = __DIR__.'/../../Fixtures/project/bin/console';
+    protected static $frontControllerPath = __DIR__.'/../../Fixtures/project/web/index.php';
```

Comment:
I'd prefer to have these assignments aligned... It will be more readable.

Can we also replace these static properties with simple class properties?